File: Atividade2/FuncionarioJ/FuncionarioP/FuncionarioRegular.py
from Funcionario import Funcionario #Importando classe do arquivo

class FuncionarioRegular(Funcionario):
    def __init__(self, nome, id, salario_base):
        super().__init__(nome, id, salario_base)

    def calcular_bonus(self) -> float:
        if self.salario_base <= 5000:
            return self.salario_base * 0.10
        elif self.salario_base <= 10000:
            return self.salario_base * 0.15
        else:
            return 0
        
        

File: Atividade2/FuncionarioJ/FuncionarioJ/FuncionarioRegular.java
package FuncionarioJ;
public class FuncionarioRegular extends Funcionario{

    public FuncionarioRegular(String nome, int id, double salario_base){
        super(nome, id, salario_base);
    }

    @Override
    public double calcularBonus(){
        double salario_base = getSalarioBase();

        if(salario_base <= 5000){
            return salario_base * 0.10;
        } else if (salario_base > 5000 && salario_base <= 10000) {
            return salario_base * 0.15;
        }else
            return 0;
    }
}